files/migrations/0022_remove_media_media_user_state_date_idx_and_more.py

```
@@ -0,0 +1,21 @@
+    ]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name='media',
+            name='media_user_state_date_idx',
+        ),
+        migrations.RemoveIndex(
+            model_name='media',
+            name='media_user_encoding_idx',
+        ),
+    ]
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)